File: C.C++/A1/A1-018.cpp
#include <iostream>
using namespace std;

int main()
{
    int n;
    
    // input
    cin>> n;

    // process

    if (n >= 1 && n <= 9)
    {
        if (n <= 3) {
            for (int i = 0; i < n; i++)
            {
                // output

                cout<< "I";
            }
        } 
        else if (n == 4)
        {
            cout<< "IV";
        }
        else if (n == 9)
        {
            cout<< "IX";
        }
        else 
        {
            cout<< "V";
            for (int i = 0; i < n - 5; i++)
            {
                cout<< "I";
            }
        }
    } else if (n < 0)
    {
        cout<< "Error : Please input positive number";
    } else
    {
        cout<< "Error : Out of range";
    }
}

File: Python/A1/A1-018.py
# Author: Korawit Suporn
# Date  : 23 Apr. 2025
# Editor: Vim

n = int(input())

if n > 9 or n == 0:
    print("Error : Out of range")
elif n < 0:
    print("Error : Please input postive number")
else:
    while n:
        if n == 9:
            print('IX', end = "")
            n -= 9

        elif n >= 5:
            print('V', end = "")
            n -= 5

        elif n == 4:
            print("IV", end = "")
            n -= 4
        else:
            print('I', end = "")
            n -= 1
